project-p2-layouts/code/index.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>FoodBlog</title>
+    <link rel="stylesheet" type="text/css" href="style.css">
+  </head>
+  <body>
+    <div class="header">
+      <h1 class="top-header">Food Blog</h1>
+    </div>
+    <div class="menu">
+      <a href="overview.html">Noodles</a>
+      <a href="overview.html">Rice</a>
+      <a href="overview.html">Meat</a>
+      <a href="overview.html">Vegi</a>
+    </div>
```

Comment:
We should try to use semantic HTML elements when we can, so look into the header element and the nav element.